vpd-tool/src/vpd_tool_main.cpp

```diff
@@ -100,6 +123,61 @@ int main(int argc, char** argv)
         l_rc = l_vpdToolObj.readKeyword(l_vpdPath, l_recordName, l_keywordName,
                                         l_isHardwareOperation, l_filePath);
     }
+    else if (!l_writeFlag->empty())
```

Comment:
what if the user fails to provide either --value or --file ?  something like below
vpd-tool -w -O < > -R < > -K < > 

there is no check to tell the user that one of the above two options is required for writes. can you please add that.

Reply:
line no 170 covering this.

> else if (l_fileOption->empty() && l_keywordValueOption->empty())